### DISCUSSION TITLE
UI issue The environments variables under settings is locked down I have no idea why

### DISCUSSION THREAD

**Discussion by lesliewubbel** [vercel/community#3230] [Help]
### Summary

I go to my project's settings to change the environmental settings my mouse cursor turns into circle with line cross it in red. Not letting me edit the environment variables

![vercel issue](https://github.com/vercel/community/assets/13157730/4808e195-b4ac-4cc2-a791-5c3f5ce66c1c)

### Steps to Reproduce

1) log into vercel
2) click on project
3) click settings 
4) click on left side bar select environmental variables

**Comment by dmitriy-MI6** [comment]
https://github.com/orgs/vercel/discussions/3229

**Comment by lesliewubbel** [comment]
There is a duplicate issue closing now.
